cmd/helm/helm.go

```diff
@@ -55,7 +56,8 @@ func init() {
 
 func debug(format string, v ...interface{}) {
 	if settings.Debug {
-		format = fmt.Sprintf("[debug] %s\n", format)
+		timeNow := time.Now().UTC().String()
```

Comment:
This should probably use something like RFC3339 formatted timestamps instead of just the raw string implementation

Reply:
But let me check on the time format before doing anything here

Reply:
I think I'd go with RFC3339, unless you have inclinations otherwise